Cargo.toml

```diff
@@ -35,3 +35,6 @@ version-sync = "~0.7"
 # Some tests are rather slow. Furthermore, optimalisations tend to provoke races and UBs to
 # manifest, so we want to try that in tests if possible.
 opt-level = 1
+
+[package.metadata.docs.rs]
+all-features= true
```

Comment:
It's weird it's not showing the blank line 41 here, I see it in my editor

Reply:
I guess it's because your editor considers `\n` a line *separator*, while github (and other editors) consider `\n` line *terminator*. Your file ends with `\n`, which for github terminates line 40, for your editor it separates line 40 from 41 (that is empty).

Previously, you didn't have the `\n` at the very end so github showed some red weird icon that meant it is missing the end of the last line.